apps/ui/src/views/Proposal/Overview.vue

```diff
@@ -46,12 +46,18 @@ const cancelling = ref(false);
 const aiSummaryOpen = ref(false);
 
 const editable = computed(() => {
+  if (!compareAddresses(props.proposal.author.id, web3.value.account))
+    return false;
+
+  if (offchainNetworks.includes(props.proposal.network)) {
+    return props.proposal.start > Math.floor(Date.now() / 1000);
+  }
+
```

Comment:
This used to work for offchain spaces.

Do we need to have two ways of handling it? Ideally we should look why it doesn't work anymore and here don't fragment it further.